.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: daily
```

```suggestion
    interval: monthly
```

Monthly means things don't get too out of date, and will be less noisy.

Docs:

* https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates#frequency-of-dependabot-pull-requests

See also:
 
* https://github.com/python/bedevere/pull/497
* https://github.com/python/devguide/pull/922
* https://github.com/python/miss-islington/pull/573